apps/remix-ide/src/app/tabs/settings-tab.js

```diff
@@ -186,6 +207,13 @@ module.exports = class SettingsTab extends ViewPlugin {
       self.config.set('settings/personal-mode', !isChecked)
       elementStateChanged(self._view.personalLabel, isChecked)
     }
+    function onchangeMatomoAnalytics () {
+      const isChecked = self.config.get('settings/matomo-analytics')
+
+      self.config.set('settings/matomo-analytics', !isChecked)
```

Comment:
you set 'settings/matomo-analytics' ok , but where do you actually say matomo to stop?

Reply:
I disable this from UI. I'm not sure it is possible. I will work on this once we have it working